### ISSUE TITLE
ADD-IN error when try to add it to Excel Windows

### ISSUE THREAD

**Issue by hwasiti** [Emurasoft/excel-csv-import#89]
I am getting this error when I click on trust this addon in Excel 2016 on Windows 10:
Excel Add-in error
we can't load this add-in because we couldn't connect to the catalog

I tried the same steps on Mac and it worked. Any other method to add this addon to my Windows Excel?

Do you have the compiled file that I can download and manually add it? 
I have added some add-ins before manually this way. Appreciate if you have the compiled file. 
I don't have the expertise to compile it from the source code.

**Comment by MakotoE** [comment]
Try using Excel Online by going to https://www.office.com/, signing in, and opening the Excel app. Add the add-in from there.

You mentioned that you added add-ins manually; is this because you got the same error in other add-ins as well?

**Comment by hwasiti** [comment]
`Try using Excel Online by going to https://www.office.com/, signing in, and opening the Excel app. `
That should work, but I prefer the offline excel for large data sheets.

` is this because you got the same error in other add-ins as well?`

Not really.. I remember that addon has only this manual method and not through the Microsoft store.

I think I got it from [here](https://jaimonmathew.wordpress.com/2011/08/23/excel_addin_to_work_with_unicode_csv/)

Your add-in is better since it allows me to customize the separator as well.

**Comment by MakotoE** [comment]
If you want to add this add-in yourself, you'll have to host it on your localhost, as described in [this tutorial](https://docs.microsoft.com/en-us/office/dev/add-ins/testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins). That means I can't just share the compiled files, you will need some kind of http server.

Sorry but I don't have Excel 2016 so I am not supporting that version.

If you have a large data sheet, I would export it as either a CSV file and open it with a text editor, or export it to a SQL database.

**Comment by hwasiti** [comment]
`Sorry but I don't have Excel 2016 so I am not supporting that version.`

In the mac where it worked the Excel was v.2019. So indeed maybe the issue with Excel 2016.

`If you have a large data sheet, I would export it as either a CSV file and open it with a text editor, or export it to a SQL database.`
Eventually, I went through that route. I was just a bit lazy and wanted to do it in Excel :)
I used pandas and python which was super fast. But I was still interested in your add-in for the future. When I will update to Excel 2019, maybe it will work.

Thanks for your replies and support.
